internal/tools/aws/helpers_sql.go

```diff
@@ -43,13 +43,18 @@ func MattermostPostgresConnStrings(schema, username, password string, dbCluster
 
 // MattermostPostgresPGBouncerConnStrings formats the connection strings used by
 // Mattermost servers to access a PostgreSQL database with a PGBouncer proxy.
-func MattermostPostgresPGBouncerConnStrings(username, password, database string) (string, string) {
-	dbConnection := fmt.Sprintf("postgres://%s:%s@pgbouncer.pgbouncer:5432/%s?connect_timeout=10&sslmode=disable",
+//
+// Regarding binary_parameters:
+// https://blog.bullgare.com/2019/06/pgbouncer-and-prepared-statements
+func MattermostPostgresPGBouncerConnStrings(username, password, database string) (string, string, string) {
+	dbConnection := fmt.Sprintf("postgres://%s:%s@pgbouncer.pgbouncer:5432/%s?connect_timeout=10&sslmode=disable&binary_parameters=yes",
+		username, password, database)
+	readReplicas := fmt.Sprintf("postgres://%s:%s@pgbouncer.pgbouncer:5432/%s-ro?connect_timeout=10&sslmode=disable&binary_parameters=yes",
```

Comment:
Strictly speaking, you don't need this for reading. But fine to keep it consistent. Maybe good to refactor the entire thing as a net.URL and stringify it to reduce duplication. 0/5